src/pushsource/_impl/backend/errata_source/errata_client.py

```diff
@@ -73,49 +110,193 @@ def get_raw_f(self, advisory_id):
         )
         return f_map(all_responses, lambda tup: ErrataRaw(*tup))
 
-    def _call_et(self, method_name, advisory_id):
-        method = getattr(self._errata_service, method_name)
-
+    def _call_et(self, method, advisory_id):
         # These APIs have had performance issues occasionally, so let's set up some
         # detailed structured logs which can be used to check the performance.
         LOG.debug(
             "Calling Errata Tool %s(%s)",
-            method_name,
+            method,
             advisory_id,
             extra={
                 "event": {
                     "type": "errata-tool-call-start",
-                    "method": method_name,
+                    "method": method,
                     "advisory": advisory_id,
                 }
             },
         )
         try:
-            out = method(advisory_id)
+            out = self._do_call(method, advisory_id)
+
             LOG.debug(
                 "Errata Tool completed call %s(%s)",
-                method_name,
+                method,
                 advisory_id,
                 extra={
                     "event": {
                         "type": "errata-tool-call-end",
-                        "method": method_name,
+                        "method": method,
                         "advisory": advisory_id,
                     }
                 },
             )
+
             return out
         except:
             LOG.debug(
                 "Failed to call Errata Tool %s(%s)",
-                method_name,
+                method,
                 advisory_id,
                 extra={
                     "event": {
                         "type": "errata-tool-call-fail",
-                        "method": method_name,
+                        "method": method,
                         "advisory": advisory_id,
                     }
                 },
             )
             raise
+
+
+class ErrataClient(ErrataClientBase):
+    def __init__(self, threads, url, **retry_args):
+        deprecation_notice = """
+               The XML RPC Errata Client has been deprecated and will be removed.
+               Please see <some link> for more information on how to transition to
```

Comment:
The "some link" was meant to be a placeholder which would be pointed at some actual doc we'd write about this.

That being said though, maybe the situation is not really complicated enough to warrant that.  I guess the instructions are simple enough to go in a single sentence? Here's my attempt:

```
The XML RPC Errata Client has been deprecated and will be removed.
Please provide a `keytab_path` and `principal` to the errata pushsource backend.
```